src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -291,7 +291,7 @@ def __init__(self, target_path, strategy_dict=None):
 
     # If we have already generated a strategy dict, use that in favor of
     # creating a new pool and picking randomly.
-    if strategy_dict:
+    if strategy_dict is not None:
```

Comment:
slightly confused here, since strategies = {} is not None, won't it go inside the loop?

Reply:
That's the intention!

Previously, if we pass "{}", it will go into the "else", which is picking strategies from scratch. 

We don't want this -- we want "{}" to mean "no strategies are picked".

Reply:
Gotcha LGTM!